semantic error: Assignment to an input function parameter `n` is not allowed
 --> tests/errors/func_02.py:4:5
  |
4 |     n = 5
  |     ^ Use InOut[n] to allow assignment
